src/Event Controllers/main.blp

```diff
@@ -23,7 +23,7 @@ Adw.StatusPage {
       }
 
       Gtk.Button ctrl_button {
-        label: _("Ctrl + Click to Activate");
+        label: _("Ctrl Key");
```

Comment:
I think the label here should indicate that it represents the Control key state, and probably shouldn't be a button anymore since the signal has been removed.